usr/src/uts/common/sys/zone.h

```diff
@@ -123,19 +129,24 @@ extern "C" {
 #define	ZONE_ATTR_INITNAME	9
 #define	ZONE_ATTR_BOOTARGS	10
 #define	ZONE_ATTR_BRAND		11
-#define	ZONE_ATTR_SCHED_CLASS	12
-#define	ZONE_ATTR_FLAGS		13
-#define	ZONE_ATTR_HOSTID	14
-#define	ZONE_ATTR_FS_ALLOWED	15
-#define	ZONE_ATTR_NETWORK	16
-#define	ZONE_ATTR_DID		17
-#define	ZONE_ATTR_INITNORESTART	18
-#define	ZONE_ATTR_APP_SVC_CT	19
-#define	ZONE_ATTR_SCHED_FIXEDHI	20
+/*#define	ZONE_ATTR_PHYS_MCAP	12*/
```

Comment:
cstyle will complain about this - we had to go back and add spaces.
```
zone.h: 130: missing blank after open comment
zone.h: 130: missing blank before close comment
```

See https://github.com/omniosorg/illumos-omnios/pull/1156

Reply:
Damn, I did `git am` on both, I swore I did.  `git rebase -i` must've blown it off or something.  :(